records/2021-10-02-buy-poodle.md

```diff
@@ -0,0 +1,25 @@
+# Buy A Poodle
+
+## Status
+<!-- What is the status, such as proposed, accepted, rejected, deprecated, superseded, etc.? -->
+Proposed
+
+## Context
+<!-- What is the issue that we're seeing that is motivating this decision or change? -->
+I've been lonely lately, especially given all the lock-downs and restrictions during COVID-19.
+
+I like cats but dogs are more fun. Dog poo doesn't stink as much too.
```

Comment:
Could you elaborate on why dogs are more fun? 

Don't forget that wet dogs are really stinky too...